### PR TITLE
Review request

### PR DESCRIPTION
**Pull request by abdellani**

### CODE REVIEW

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Hello, devs!! 😁 Welcome to the JavaScript section of the curriculum :clap::tada:

Here are my initial observations for this project. I made some comment as well please check them out. They are optional by the way.

- Your code is failing the linter check. Please check the details to see how to fix it. It is important to get a pass on this project and it also makes your code look professional :grin:
- Your `new book` button does not bring up the `new book form`. Please check and fix this. It's advisable to always check your console as well while debugging to pin down where a problem is coming from. 

Screenshot 2019-09-17 at 11 02 54 AM

- Your `new book` form should also contain a toggle to indicate whether a book has been read or not.

**Please fix those and request for a re-review.**

**Status**: `Changes required`

**Happy coding!**

File: javascript/javascript.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    && document.getElementById('book-pages').value !== '') {
+    addBookToLibrary();
+  }
+});
+
+function addBookToLibrary() {
+  title = document.getElementById('book-title').value;
+  author = document.getElementById('book-author').value;
+  pages = document.getElementById('book-pages').value;
+  newBook = new Book(title, author, pages);
```
[the remark this answers is not in the document]
Comment (EmirVatric) [reply javascript/javascript.js]: We have implemented check if all fields are populated on line 24, on button event listener.

**Comment by abdellani** [comment]
Hi @JamesOkunlade 
Thank you for the review.
The error from sticker says that there is an unexpected token ')'. But we checked our code and we didn't find anything wrong. We also used a character count to check the number of the opening and closing parenthesizes, and we found that they are perfectly balanced. 

image

**Review by BalintMendli — APPROVED** [review]
Hi @abdellani @EmirVatric ,

Nice job, the app works great.

This is a 👍 

You can go ahead and merge the project.

[Optional] Consider adding some styling to the app to make it look nicer.

**Pull request merged**